app-web/src/components/PrimaryHeader/PrimaryHeader.js

```diff
@@ -1,60 +1,17 @@
 import React from 'react';
-import PropTypes from 'prop-types';
-import { connect } from 'react-redux';
-
 import classes from './PrimaryHeader.module.css';
 
 import { APP_TITLE } from '../../constants/strings';
 import Banner from '../Common/Banner';
-import Button from '../UI/Button/Button';
-import { LOGOUT_BTN_ID, LOGIN_BTN_ID } from '../../constants/ui';
-import implicitAuthManager from '../../auth';
-
-export const PrimaryHeader = ({ isAuthenticated }) => {
-  let button = (
-    <Button
-      type="primary"
-      id={LOGIN_BTN_ID}
-      clicked={() => {
-        window.location = implicitAuthManager.getSSOLoginURI();
-      }}
-    >
-      Login
-    </Button>
-  );
-  if (isAuthenticated) {
-    button = (
-      <Button
-        type="primary"
-        id={LOGOUT_BTN_ID}
-        clicked={() => {
-          implicitAuthManager.clearAuthLocalStorage();
-          window.location = implicitAuthManager.getSSOLogoutURI();
-        }}
-      >
-        Logout
-      </Button>
-    );
-  }
+import Login from '../Auth/Login/Login';
```

Comment:
Personally I think its best to stick with lower case paths were possible. They can be finicky in different environments: Mac does no care; Linux cares; Windows does not care.